discretize/TensorMesh.py

```diff
@@ -160,6 +160,24 @@ def gridN(self):
         return self._getTensorGrid('N')
 
     @property
+    def gridH(self):
```

Comment:
@dccowan: do you have an example of where this is used? Some context would be helpful. 

A couple comments: this isn't really a `grid` in the same sense as we are using the word elsewhere in the codebase, so we should think about the naming here. `grid` is really used to denote locations in space (spatial locations of cell centers, nodes, faces, etc), whereas this is a tensor representation of the cell widths. 

I am also a bit hesitant to add something to the class that is really a view of information that we already have accessible through `mesh.h`. This is where having an example and a bit of context would be quite helpful.